source/systems/common.rst

```diff
@@ -64,5 +64,1789 @@ RDHPCS Platforms
+are simple perl scripts that are located at:
+
+.. code-block:: shell
+
+   /contrib/sutils/bin/arbitrary.pl
+   /contrib/sutils/bin/layout.pl
+
+
+Please feel free to copy these scripts and modify them to suit your needs.
+
+MPMD example
+""""""""""""
+
+Another script called ``layout.pl`` is available when sutils module is
+loaded that can be used for MPMD jobs.
+
+The script ``layout.pl`` has a syntax similar to &quot;mpirun&quot; and it
+creates a configuration file that is used with --multi-prog option of the srun
+command.
+
+It is best illustrated with a simple example:
+
+Suppose we want to run on 12 nodes, using 10 tasks per node, and would like to
+launch:
+
+* 20 tasks of a.out
+* 60 tasks of b.out
+* 40 tasks of c.out
+
+.. code-block:: shell
+
+   sbatch --nodes=12 --ntasks-per-node=10 … slurmjob
+
+   module use -a /contrib/sutils/modulefiles
+   module load sutils
+   layout.pl -n 20 a.out : -n 60 b.out : -n 40 c.out &gt; lay2-$SLURM_JOB_ID
+   srun --multi-prog lay2-$SLURM_JOB_ID
+
+Specifying Wall Clock Time
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You should specify a wall clock time for your job. If you do not set a wall
+clock time it will **default to 5 minutes**. **We recommend that you do NOT set
+a wall clock time less than 5 minutes**. If your jobs will take longer than 5
+minutes, request a wall clock time reasonably close to but not less than (see
+note below) the actual wall clock time that the job will take to run.
+Specifying an excessively large wall clock time will result in increased wait
+time for your job to start (qwait), and more importantly reduced scheduler
+efficiency and overall system utilization. When requesting Multiple Partitions
+(see below), as is recommended, take into account the longest run time
+partition. Due to several other factors that effect run time your job run time
+on a &quot;slower&quot; partition may be better as compared to the Billable
+TRes per Core Performance Factor listed in the Partition tables above.
+Therefore:
+
+**Frequently** review the wall clock time of the jobs you run in order to
+better estimate your requested wall clock time. Increased accuracy of specified
+wall clock time with your job submissions will shorten queue wait times, and
+increase scheduler efficiency and overall system utilization.
+
+.. note::
+
+   Any job that runs longer than its requested wall clock time or the partition's
+   time limit will be terminated by the scheduler. When specifying your wall clock
+   time, add some extra time to your recent observed run time history to be sure
+   it will finish: **>10-20%</span>** for short run times, **>5-10%</span>** for
+   long run times, to allow for random fluctuations in run times caused by system
+   load.
+
+For example, to set a one-hour time limit:
+
+.. code-block:: shell
+
+   #SBATCH --time=1:00:00
+
+For the maximum wall clock allowed see the Queue(QOS) tables below.
+
+Specifying a Quality of Service (QOS) - Jet and Hera
```

Comment:
How to specify the QOS is (or should be) on the slurm page.  Other hera/jet information should be in the hera/jet user guides.